Changed the default temporary system dir used from ps3mediaserver to ps3mediaserver-$(USER)
--- a/net/pms/configuration/TempFolder.java
+++ b/net/pms/configuration/TempFolder.java
@@ -12,11 +12,11 @@
  * 
  * @author Tim Cox (mail@tcox.org)
  */
 class TempFolder {
 
-	private static final String DEFAULT_TEMP_FOLDER_NAME = "ps3mediaserver";
+	private static final String DEFAULT_TEMP_FOLDER_NAME = "ps3mediaserver-" + System.getProperty("user.name");
 
 	private final String userSpecifiedFolder;
 	
 	private File tempFolder;
 
